docs/Displaying a User Interface/Template Subscription Buttons/index.md

```diff
@@ -89,6 +89,24 @@ sdlManager.sendRPC(subscribeButtonRequest);
 ```
 !@
 
+@![javascript]
+```js
+sdlManager.addRpcListener(SDL.rpc.enums.FunctionID.OnButtonPress, function (onButtonPress) {
+    if (onButtonPress instanceof SDL.rpc.RpcNotification) {
```

Comment:
Don't you want to check if it's the _correct_ RpcNotification (that it's actually an `OnButtonPress` notification?